### PR TITLE
Update index.rst

### PR DESCRIPTION
**Pull request by Foadsf**
to avoid multiple errors and warnings

### CODE REVIEW

File: readthedocs/index.rst [hunk 1]
```diff
@@ -4,9 +4,21 @@ Telethon's Documentation
 
 .. code-block:: python
 
+   import sys
+   import os
+   os.chdir(sys.path[0])
```
Comment (Lonami) [inline readthedocs/index.rst:9]: Why is this needed, and what path is exactly that?

File: readthedocs/index.rst [hunk 2]
```diff
@@ -4,9 +4,21 @@ Telethon's Documentation
...
+   
+   session_name = "name"
+   
+   if f"{session_name}.session" in os.listdir():
+      os.remove(f"{session_name}.session")
```
Comment (Lonami) [inline readthedocs/index.rst:14]: Constantly having to login again is not something we want to teach (and that's a bad way to check if a file exists).

File: readthedocs/index.rst [hunk 3]
```diff
@@ -4,9 +4,21 @@ Telethon's Documentation
...
+      
+   import nest_asyncio
```
Comment (Lonami) [inline readthedocs/index.rst:16]: This module is not in the Python standard library. What is it, where does it come from, and what does it do?

File: readthedocs/index.rst [hunk 4]
```diff
@@ -4,9 +4,21 @@ Telethon's Documentation
...
+   nest_asyncio.apply()
+
    from telethon.sync import TelegramClient, events
 
-   with TelegramClient('name', api_id, api_hash) as client:
+   async with TelegramClient(session_name, api_id, api_hash) as client:
```
Comment (Lonami) [inline readthedocs/index.rst:21]: `async with` just won't work outside `async def` unless it's IPython or similar.

**Comment by Foadsf** [comment]
@Lonami Thanks for the replies. To get the rationalte behind the edits please try to follow the tutorial as a beginner and follow along the erros and warning. Though I should say that this PR is not good and I should send another.

File: readthedocs/index.rst (hunk 1 above)
Comment (Foadsf) [reply readthedocs/index.rst:9]: It is to set the working directory to the script's path and later delete the `.session` file

File: readthedocs/index.rst (hunk 2 above)
Comment (Foadsf) [reply readthedocs/index.rst:14]: please see [this](https://stackoverflow.com/a/60978983/4999991)

File: readthedocs/index.rst (hunk 3 above)
Comment (Foadsf) [reply readthedocs/index.rst:16]: see [this](https://stackoverflow.com/a/56434301/4999991)

File: readthedocs/index.rst (hunk 4 above)
Comment (Foadsf) [reply readthedocs/index.rst:21]: I indeed tested the code in Jupyter

File: readthedocs/index.rst (hunk 3 above)
Comment (Lonami) [reply readthedocs/index.rst:16]: I have left a comment. My point remains, we should not really teach this.

Comment (Foadsf) [reply readthedocs/index.rst:16]: @Lonami Thanks for the comment, and I replied. Do you have a better solution for that recurring problem? Basically, when you want to test your code and have to rerun multiple times,this happens indefinitely. I'm not a sqlite3 expert, but there might be a way to check if that file already exists; just use it in some way.

**Comment by Lonami** [comment]
I will accept a different PR adding a `.. note::` in the [Editing the code - Signin In](https://docs.telethon.dev/en/latest/basic/signing-in.html#editing-the-code) section with a mention to Jupyter and IPython, something like:

```rst
.. note::

    Jupyter and IPython run the `asyncio` event loop for you, which means some of the magic in Telethon won't work as is. Simply replace `with` with `async with` and add `await` when using the methods instead of using the loop manually.
```

**Comment by Lonami** [comment]
Maybe you can also create a separate `.rst` file that documents how to properly use Telethon with `asyncio` if that note gets too large or requires more explanation. Or just recommend to use normal `python` shell and, once the user is used to it, go back to IPython.

**Comment by Foadsf** [comment]
@Lonami I will work on your suggestions. in the meantime I posted a new question [here](https://stackoverflow.com/q/60989409/4999991) summarizing my experience and challenges running the code snippet.
